### DISCUSSION TITLE
Maybe consider hyprshade over wlsunset

### DISCUSSION THREAD

**Discussion by clsty** [end-4/dots-hyprland#202] [Ideas]
I've discovered [hyprshade](https://github.com/loqusion/hyprshade) recently, and it provides more freedom for customization, not only just modify gamma.

Note: Screenshot will be affected by screen shaders too, but there're some workarounds [here](https://github.com/loqusion/hyprshade/issues/22).

**Comment by end-4** [comment]
wlsunset changes ~~gpu temperature~~color temperature (according to what ive heard),  while shaders apply extra effect.
So wlsunset should be more power efficient

I could be wrong

**Reply by clsty** [reply]
Wow, that's really interesting. I'm wondering how they achive this (GPU temp.).

**Reply by clsty** [reply]
I searched for it online, and it appears to be color temperature, no info about GPU temperature. Anyway I guess wlsunset to GPU temperature is like, they're just different things?

**Reply by end-4** [reply]
my bad. color temperature.
someone mentioned gpu when they told me this but i can't remember exactly
